README.md

```diff
@@ -1,52 +1,60 @@
 
-* :rocket: **Integration with other Nextcloud apps!** Currently Contacts – more to come.
+* :rocket: **Integration with other Nextcloud apps!** Currently Contacts and Circles – more to come.
 * :globe_with_meridians: **WebCal Support!** Want to see your favorite team's matchdays in your calendar? No problem!
 * :raising_hand: **Attendees!** Invite people to your events.
-* :see_no_evil: **We’re not reinventing the wheel!** Based on the great [davclient.js](https://github.com/evert/davclient.js), [ical.js](https://github.com/mozilla-comm/ical.js) and [fullcalendar](https://github.com/fullcalendar/fullcalendar) libraries.
+* :alarm_clock: **Reminders!** Get alarms for events inside your browser and via email.
+* :see_no_evil: **We’re not reinventing the wheel!** Based on the great [c-dav library](https://github.com/nextcloud/cdav-library), [ical.js](https://github.com/mozilla-comm/ical.js) and [fullcalendar](https://github.com/fullcalendar/fullcalendar) libraries.
 
 And in the works for the [coming versions](https://github.com/nextcloud/calendar/milestones/):
 * :mag: Search for events ([#8](https://github.com/nextcloud/calendar/issues/8))
-* :alarm_clock: Get alarms for events inside your browser ([#51](https://github.com/nextcloud/calendar/issues/51))
 * :watch: See when other attendees are free ([#39](https://github.com/nextcloud/calendar/issues/39))
 
-## Installation
+## :hammer_and_wrench: Installation
 
 In your Nextcloud, simply navigate to »Apps«, choose the category »Organization«, find the Calendar app and enable it.
 Then open the Calendar app from the app menu.
 
-## Support
+## :satellite: Support
 
-Check out our [FAQ](https://github.com/nextcloud/calendar/wiki/FAQs). If you dont find a solution, you are welcome to [ask for support](https://help.nextcloud.com) in our Forums or the IRC-Channel. If you have found a bug, feel free to open a new Issue on GitHub. Keep in mind, that this repository only manages the frontend. If you find bugs or have problems with the CalDAV-Backend, you should ask the guys at [Nextcloud server](https://github.com/nextcloud/server) for help!
+If you need assistance or want to ask a question about Calendar, you are welcome to [ask for support](https://help.nextcloud.com/c/apps/calendar) in our Forums or the [IRC-Channel](https://webchat.freenode.net/?channels=nextcloud-calendar).
+If you have found a bug, feel free to open a new Issue on GitHub. Keep in mind, that this repository only manages the frontend.
+If you find bugs or have problems with the CalDAV-Backend, you should ask the team at [Nextcloud server](https://github.com/nextcloud/server) for help!
 
-## Supported Browsers
+## :earth_africa: Supported Browsers
 
-* Chrome/Chromium 49+
-* Edge 14+
-* Firefox 45+
+* Chrome/Chromium 76+
+* Edge 40+
+* Firefox 60+
 * Internet Explorer 11
-* Safari 10+
+* Safari 12.1+
 
 We don't support Internet Explorer 10 and below. Patches for IE9+ are accepted though.
```

Comment:
Maybe just remove the second sentence.

Reply:
Then I can also remove both sentences, since it should be obvious that we don't support IE 10 if the list says IE11